lib/identity-idp-functions/mock_proofers/address_mock.rb

```diff
@@ -0,0 +1,31 @@
+require 'proofer'
+
+module IdentityIdpFunctions
+  module MockProofers
+    class AddressMock < Proofer::Base
+      vendor_name 'AddressMock'
```

Comment:
Ok so I *think* this may cause issues --- every directory under `source/` is deployed as its own folder (hence their own Gemfile, etc) so having this logic get pulled in from outside the `source/` may not work ... I think we need to move this inside.

I *think* the `aws-ruby-sdk` folder is an example of sharing code across multiple lambdas, so if we did want to share this in multiple lambdas, I think we'd have to do something like that and update the `template.yaml` to include things appropriately